audit-ci.json

```diff
@@ -1,5 +0,0 @@
-{
-  "low": true,
-  "skip-dev": true,
```

Comment:
Should we only audit production dependencies (to provide like for like behaviour). This could be achieved by running the following command `yarn npm audit --environment production --no-deprecations`

Reply:
I think we should audit dev deps, if there's a vuln in one of the packages and they're running on our dev machines, it would make sense

Reply:
I agree, thanks!